File: Python/0008. Standings.py
# If you need to import additional packages or classes, please import here.
import math
def func():
    # please define the python3 input here. 
    # For example: a,b = map(int, input().strip().split())
    # please finish the function body here.
    # please define the python3 output here. For example: print().
    
    while True:
        try:
            m = int(input())
            m_sq = int(math.sqrt(m))
            if m_sq*m_sq != m :
                res = "" 
                n = 0
            else: 
                res = str(m_sq)
                n = 1
            for x in range(m_sq,0,-1):
                if m % x == 0 and x * x != m:
                    res = str(x) + " " + res + " " + str(m//x)
                    n += 2
            res = res.replace("  "," ")
            print(str(n) + " " + res)
            
            
            
        except EOFError:
            break

    
if __name__ == "__main__":
    func()

File: C++/0008. Standings.cpp
// we have defined the necessary header files here for this problem.
// If additional header files are needed in your program, please import here.
#include<math.h>
int main()
{  
  // please define the C++ input here. For example: int a,b; cin>>a>>b;;  
  // please finish the function body here.  
  // please define the C++ output here. For example:cout<<____<<endl;
    int n;
    int num = 0;
    while(cin>>n){
        int sq = floor(sqrt(n));
        string res;
        if(sq*sq==n){
            res = to_string(sq);
            num = 1;
        }
        for(int i=sq;i>0;--i){
            if(n%i==0 and i*i!=n){
                if (res == ""){
                    res = to_string(i)+" "+to_string(int(n/i));
                }
                else{
                    res = to_string(i)+" "+res+" "+to_string(int(n/i));
                }
                num += 2;
            }
        }
        cout<<num<<" "<<res<<endl;
    }
    return 0;
}
